### PR TITLE
WIP - Add the first resource for IT, a bucket for the wiki archive

### PR DESCRIPTION
**Pull request by kookster**

### CODE REVIEW

File: it/storage/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Storage Stack
+
+This template is used for the storage stacks used for IT purposes
+
+## S3 Buckets
+
+There are many S3 buckets used as part of the CI/CD that are defined within CloudFormation templates, and don't need to be managed separately. A few buckets, though, must exist prior to launching any parts of this system, and must be configured correctly. The following describe each of those buckets:
```
Comment (farski) [inline it/storage/README.md]: The buckets described by this template aren't prerequisites for CI or CD

File: it/storage/storage.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: >
+  Creates the standard S3 buckets that are necessary for launching the rest of
+  the infrastructure
```
Comment (farski) [inline it/storage/storage.yml]: Not necessary for launching other parts of Infrastructure

File: it/storage/README.md (hunk 1 above)
Comment (kookster) [reply it/storage/README.md]: yeah, this was not ready for review, I was showing @debenedictis how to work with the infrastructure repo, and how to start a PR by opening it. I'll do some work on it with him, and hopefully then we'll be ready.

**Comment by kookster** [comment]
ok @debenedictis I have this updated a bit, we should spend some more time on it when you have a chance - maybe pull in @klbaradan too...

**Comment by debenedictis** [comment]
I believe this has been completed.  Details are in Asana:
https://app.asana.com/0/893496666545006/805554352634385/f

**Comment by farski** [comment]
@debenedictis This pull request is still listed as a WIP and hasn't been merged

**Comment by kookster** [comment]
clearly, this isn't getting finished, so closing

**Pull request closed without merging**